### DISCUSSION TITLE
Trigger hover for Dropdown Menu

### DISCUSSION THREAD

**Discussion by goustaff** [radix-ui/themes#139] [Ideas]
I'd like to create a discussion to follow up this issue: https://github.com/radix-ui/themes/issues/118

@vladmoroz answered:
> Dropdown Menu isn't supposed to open on hover. It's not a typical design pattern that we'd plan to support.
> 
> Our recommendation is to re-consider your design so that it opens on click, or look into components that are designed to open on hover out of the box:
> 
> * [Navigation Menu](https://www.radix-ui.com/primitives/docs/components/navigation-menu)
> * [Hover Card](https://www.radix-ui.com/primitives/docs/components/hover-card)
> * [Tooltip](https://www.radix-ui.com/primitives/docs/components/tooltip)

But, _Dropdowns_ offer a nice enough implementation to nest several of them, whereas it doesn't seem to be the usecase of the Navigation Menu?

I think it's a feature that's desirable.
It could be easy to use with a `withHover` flag on `<DropdownMenu />`, or something similar.

**Comment by vladmoroz** [comment]
You can nest multiple submenus in Navigation Menu primitive, refer to the example here:
https://www.radix-ui.com/primitives/docs/components/navigation-menu#with-submenus

And as I said, hover to open isn't an idiomatic dropdown menu interaction, so it's not something we'd realistically prioritise

**Comment by goustaff** [comment]
> And as I said, hover to open isn't an idiomatic dropdown menu interaction, so it's not something we'd realistically prioritise

Oops, sorry.
I'll give it a try!

> And as I said, hover to open isn't an idiomatic dropdown menu interaction, so it's not something we'd realistically prioritise

:(
